publishable/lang/en/seeders.php

```diff
@@ -1,6 +1,16 @@
 <?php
 
 return [
+    'categories' => [
```

Comment:
I think I'd prefer an array with "objects" of categories. Like
```
[
    [
        'name' => '...',
        'slug' => '...'
    ],
    [
        'name' => '...',
        'slug' => '...'
    ]
]
```

So one could simply change the language-files to seed another category.
This of course applies to posts/pages as well